game.py

```diff
@@ -20,7 +20,37 @@ def snowman(snowman_word):
     If the player wins and, 
     'Sorry, you lose! The word was {snowman_word}' if the player loses
     """
-    pass
+
+    game_on = True
+
+    correct_letter_guess_statuses = build_letter_status_dict(snowman_word)
+    wrong_guess_list = []
+    wrong_guess_count = 0
+
+    print("Welcome to Snowman Game!")
+    print_word_progress_string(snowman_word,correct_letter_guess_statuses)
```

Comment:
```suggestion
    print_word_progress_string(snowman_word, correct_letter_guess_statuses)
```

Missing white space after comma. Per the official Python style guide, called PEP 8, a white space should be "Immediately before a comma, semicolon, or colon".

You can read more about white spaces in Python code [here](https://peps.python.org/pep-0008/#whitespace-in-expressions-and-statements).

This same comment about missing a white space after a comma applies to all Python code, including lines 33, 40, and 41 in this project. 

It's important to comply with the official style guide so our code is neat and professional. While this project is relatively small, soon you'll be working on very large codebases and introducing seemingly small style issues means lots of inconsistencies all over the place which makes a project less readable and more difficult to maintain. 

If you have questions about the style of your Python, consulting the PEP8 guide is the best place to go to see what official recommendations are.